.github/workflows/main.yml

```diff
@@ -49,10 +49,13 @@ jobs:
               python3-flake8-quotes
               python3-pyxdg
               python3-zstandard
-              python3-toml
+              python3-tomli-w
               python3-pip
               rpm-build
               git
+        if: ${{ ! contains(matrix.container, 'rawhide') }}
```

Comment:
This line should go below `if: ${{ contains(matrix.container, 'opensuse') }}` as these packages are only for openSUSE.

Reply:
Right, the if is below the run. OMG. Now I think I get it.